PWGHF/TableProducer/candidateCreatorXic0Omegac0.cxx

```diff
@@ -1592,8 +1593,8 @@ struct HfCandidateCreatorXic0Omegac0 {
                          kfXic0Candidate.kfDcaXYPiFromXic, kfXic0Candidate.kfDcaXYCascToPv,
                          kfXic0Candidate.chi2GeoV0, kfXic0Candidate.chi2GeoCasc, kfXic0Candidate.chi2GeoXic, kfXic0Candidate.chi2MassV0, kfXic0Candidate.chi2MassCasc,
                          kfXic0Candidate.ldlV0, kfXic0Candidate.ldlCasc, kfXic0Candidate.ldlXic,
-                         kfXic0Candidate.chi2TopoV0ToPv, kfXic0Candidate.chi2TopoCascToPv, kfXic0Candidate.chi2TopoPiFromXicToPv, kfXic0Candidate.chi2TopoXicToPv,
-                         kfXic0Candidate.chi2TopoV0ToCasc, kfXic0Candidate.chi2TopoCascToXic,
+                         kfXic0Candidate.chi2NdfTopoV0ToPv, kfXic0Candidate.chi2NdfTopoCascToPv, kfXic0Candidate.chi2NdfTopoPiFromXicToPv, kfXic0Candidate.chi2NdfTopoXicToPv,
+                         kfXic0Candidate.chi2NdfTopoV0ToCasc, kfXic0Candidate.chi2NdfTopoCascToXic,
```

Comment:
This is wrong and misleading. You changed the quantity filled in the table without changing the name of the column and its getter in the data model.
This also creates a mismatch between the `HfOmegacKf` and `HfCandToXiPiKf` which both use this column, making the `treeCreatorOmegacToOmegaPi.cxx` and `treeCreatorXic0ToXiPiKf.cxx` workflow work differently using the same getter.